beast/examples/phat_small/datamodel.py

```diff
@@ -289,6 +287,5 @@ def get_obscat(obsfile=obsfile, distanceModulus=distanceModulus,
     obs: GenFluxCatalog
         observation catalog
     """
-    obs = GenFluxCatalog(obsfile, distanceModulus=distanceModulus,
-                         filters=filters)
+    obs = GenFluxCatalog(obsfile, filters=filters)
```

Comment:
No longer used in the observed catalog.  This was left over when we were putting the observed data at 10 pc (absolute mags?).  Now we put the model at the distance of the galaxy as this is needed for a number of reasons.  Figured it was best to take this out as it means *less* code to maintain.